tools/openapi-schema/ansible-ai-connect-service.yaml

```diff
@@ -580,7 +580,7 @@ paths:
           description: One or more backend services are unavailable.
 components:
   schemas:
-    ActionEnum:
+    ActionDe0Enum:
```

Comment:
Is this a typo?

Reply:
I was also confused but it's from drf_spectacular. There is two kind of `ActionEnum` and it differentiate them like this.

Reply:
Strange. I tried upgrading drf_spectacular to see if the "bug" was fixed and it just turned it to `Action4c6Enum`. :-P So I guess I'll ignore it! Thanks.